standalone/grids/dibl-polar/src/test/java/dibl/grids/polar/MatchingPolarGrid.java

```diff
@@ -0,0 +1,19 @@
+package dibl.grids.polar;
+
+public class MatchingPolarGrid extends PolarGrid
+{
+
+    public MatchingPolarGrid(int angleOnFootside, int dotsPerCircle, double diameter, double minDiameter, String fillColor, double dotSize)
+    {
+        super(angleOnFootside, dotsPerCircle, diameter, minDiameter, fillColor, dotSize);
+    }
+
+    @Override
+    double delta(double diameter, int angleOnFootside, double dotsPerCircle)
+    {
+        double angle = Math.toRadians(angleOnFootside);
+        double correction = Math.PI / (4 * dotsPerCircle);
+        correction *= (Math.tan(angle*0.93));
+        return Math.tan(angle-correction) * (diameter * Math.PI / dotsPerCircle);
+    }
+}
```

Comment:
@Veronika

Above calculations gets pretty close to matching grids. There is still a tiny moon crescent in stead of a full border on the dots, for 30 degrees on the inside, for 60 degrees on the outside. But could any lace maker put pins that accurately? 

The first correction step is mirrored in the last calculation in the second attached image. The second step is an educated guess. Especially the 93% factor could use a more theoretical approach.

Test results:
![test-result](https://cloud.githubusercontent.com/assets/10553630/7367776/45a83e62-eda2-11e4-85b5-c3d0625d95d3.png)

Theory image:
![polar-formulas](https://cloud.githubusercontent.com/assets/10553630/7366737/9efdb242-ed9a-11e4-9ec5-89460a6974f4.png)